src/Run/Setup.cs

```diff
@@ -111,7 +153,7 @@ private string BuildParametersForCommand(Dictionary<string, string> commandParam
             {
                 if (!parameters.Key.Equals("toolName") && !string.IsNullOrEmpty(parameters.Value))
                 {
-                    string value = parameters.Value.Equals("default") ? FindSettingValue(parameters.Key) : parameters.Value;
+                    string value = parameters.Value.Equals("default") ? FindSettingValue(parameters.Key) : ParseSettingValue(parameters.Value);
```

Comment:
If it is only in the default value under Settings, we don't need to call PaseSettingValue again here